models/practices-inspect/pandas-stress.ipynb

```diff
@@ -0,0 +1,155 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c1327b72-6ab0-44b7-a19a-053149ba34c2",
+   "metadata": {},
+   "source": [
+    "1. Import the `pandas` library as well as the `deepcopy` function from the `copy` standard Python package."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "30ba4f6e-a277-4741-8953-3f425b8a6522",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "from copy import deepcopy"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "31d33f31-fe25-4184-b2ed-28426fc81ff6",
+   "metadata": {},
+   "source": [
+    "2. Use Pandas to read the CSV file and store it as a data frame variable."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3fbb7708-69b1-4c32-acb5-f3d07a4b64bf",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "df = pd.read_csv('./rand_people.csv', index_col=['Initials'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "57343de0-e067-4135-9b89-3fad06fd9d39",
+   "metadata": {},
+   "source": [
+    "3. From the right-side menu, open the `Kernel Usage` panel.\n",
+    "Observe that the Jupyter kernel CPU and memory are less than ten percent and under one-hundred megabytes, respectively."
```

Comment:
At this step I have slightly a bit more than 100 MB. 

![image](https://github.com/RedHatTraining/AI26X-apps/assets/675470/7dc311a8-f4cb-4772-89cf-cd78541f76e3)

We can just tell students to verify that the memory is under 200.